{
  underTest.stop();
  expectedException.expect(ConnectException.class);
  call(underTest.getUrl());
}
